frontier_explorer/src/goal_selector.cpp

```diff
@@ -0,0 +1,119 @@
+#include <frontier_explorer/goal_selector.hpp>
+
+#include <geometry_msgs/PoseStamped.h>
+#include <visualization_msgs/MarkerArray.h>
+#include <visualization_msgs/Marker.h>
+
+namespace ariitk::frontier_explorer {
+
+FrontierComparator::FrontierComparator(const ros::NodeHandle& nh, const ros::NodeHandle& nh_private)
+    : nh_(nh)
+    , nh_private_(nh_private) {
+    nh_private_.getParam("min_frontier_size", min_size_);
+    nh_private_.getParam("clear_radius",      clear_radius_);
```

Comment:
extra space after comma

Reply:
check the complete PR again for these kind of things

Reply:
that was to actually align the param names. if that's not advised i'll edit it